pandas/tests/extension/decimal/array.py

```diff
@@ -281,6 +281,16 @@ def convert_values(param):
     def value_counts(self, dropna: bool = True):
         return value_counts(self.to_numpy(), dropna=dropna)
 
+    # We override fillna here to simulate a 3rd party EA that has done so. This
```

Comment:
So we can remove this in 3.0 when the deprecation is enforced? (if so can we add a note)

Reply:
Yes.  I'll add that to my next Assorted branch